arm-relay/2016-07-01/swagger/relay.json

```diff
@@ -0,0 +1,1930 @@
+    },
+    "HybridConnectionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HybridConnection"
+          },
+          "description": "Result of the List HybridConnection ."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of HybridConnection operation"
+        }
+      },
+      "description": "The response of the List HybridConnection operation."
+    },
+    "HybridConnection": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HybridConnectionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of HybridConnection Resource."
+    },
+    "HybridConnectionProperties": {
+      "properties": {
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the HybridConnection was created."
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the namespace was updated."
+        },
+        "listenerCount": {
+          "format": "int32",
+          "type": "integer",
+          "readOnly": true,
+          "minLength": 1,
+          "maxLength": 25,
+          "description": "The number of listeners for this HybridConnection. min : 1 and max:25 supported"
+        },
+        "requiresClientAuthorization": {
+          "type": "boolean",
+          "description": "true if client authorization is needed for this HybridConnection; otherwise, false."
+        },
+        "userMetadata": {
+          "type": "string",
+          "description": "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."
+        },
+        "path": {
+          "type": "string",
+          "description": "The path of the HybridConnection."
+        }
+      },
+      "description": "Properties of the HybridConnection."
+    },
+    "WcfRelaysListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WcfRelay"
+          },
+          "description": "Result of the List WcfRelays ."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of WcfRelays operation"
+        }
+      },
+      "description": "The response of the List WcfRelays operation."
+    },
+    "WcfRelay": {
```

Comment:
it is a proxy resource, added a new Resource definition excluding Location property .